packages/zoe/src/zoeService/invitationQueries.js

```diff
@@ -11,10 +13,8 @@ export const makeInvitationQueryFns = invitationIssuer => {
       assert.note(err, X`Due to ${reason}`);
       throw err;
     };
-    return E.get(
-      E.get(E(invitationIssuer).getAmountOf(invitationP).catch(onRejected))
-        .value,
-    )[0];
+    const amtP = E(invitationIssuer).getAmountOf(invitationP).catch(onRejected);
+    return amtP.then(({ value }) => getCopyBagEntries(value)[0][0]);
```

Comment:
First, please prefer `E.when` to `.then` unless there's a good reason to use `.then`.

```suggestion
    return E.when(amtP, ({ value }) => getCopyBagEntries(value)[0][0]);
```
then, we could use the helper here
```suggestion
    return E.when(amtP, getInvitationAmountDetails);
```